src/set-pages-path.js

```diff
@@ -1,41 +1,38 @@
 const core = require('@actions/core')
-const axios = require('axios')
 const {ConfigParser} = require('./config-parser')
 
+function getParserConfiguration(staticSiteGenerator, path) {
+  switch (staticSiteGenerator) {
+    case 'nuxt':
+      return {
+        configurationFile: './nuxt.config.js',
+        propertyName: 'router.base',
+        propertyValue: path,
+        blankConfigurationFile: `${process.cwd()}/blank-configurations/nuxt.js`
```

Comment:
:warning: Seems like we are missing a `src/` or `dist/` path segment here, but we could avoid that by using `__dirname` instead. 🤔 

```suggestion
        blankConfigurationFile: `${__dirname}/blank-configurations/nuxt.js`
```